plugins/catalog-backend-module-opa-entity-checker-processor/README.md

```diff
@@ -0,0 +1,26 @@
+# Backstage OPA Backend Module Entity Processor
+
+When it comes to validating entries the Backend and Frontend plugins are great combo to bring validation errors to the user. However, this doesn't allow you to have the big picture of hom many entities in your catalog have fail the validation.
```

Comment:
```suggestion
Using the [OPA Entity Checker](https://github.com/Parsifal-M/backstage-opa-plugins/tree/main/plugins/backstage-opa-entity-checker#opa-entity-checker) is a great way to display and help your Backstage users know when their entity metadata is incorrect, however by using this processor it will allow you to create a better overview of entities that have validation errors by adding the annotation to them
```